src/store/trails/mutations.js

```diff
@@ -53,6 +53,7 @@ export const setBallot = (state, ballot) => {
 
 export const setUserTreasuries = (state, treasuries) => {
   state.userTreasuries = treasuries;
+  console.log("Trails.mutations.setUserTreasuries() ", treasuries.rows);
```

Comment:
I think this could also be removed

Reply:
totally

Reply:
fixed here [f6280a2](https://github.com/telosnetwork/app-telos-native/pull/125/commits/f6280a25e41c0e182dd94fbddd14dbb7d0865993)